jarbas/chamber_of_deputies/tests/test_reimbursement_view.py

```diff
@@ -22,6 +27,13 @@ def get_reimbursement(**kwargs):
     return mixer.cycle(quantity).blend(Reimbursement, search_vector=None, **kwargs)
 
 
+def create_search_vector():
+    # Running the command silently
```

Comment:
What about using a proper [docstring](https://www.python.org/dev/peps/pep-0257/) instead of a comment?

```python
def create_search_vector():
    """Running the command silently"""
    …
```